packages/api-providers/src/webb-context/bridge/bridge.ts

```diff
@@ -0,0 +1,52 @@
+import { Currency } from '../currency/currency';
+import { BridgeConfig } from '../../types/bridge-config.interface';
+import { AppConfig } from '../common';
+import { WebbCurrencyId } from '../../enums';
+import { InternalChainId } from '../../chains';
+import { CurrencyRole } from '../../types/currency-config.interface';
+
+export class Bridge {
```

Comment:
This is OK, the directory should be changed to `anchor/bridge.ts` because ultimately this is a bridge between anchors.